Classes/DDLog.m

```diff
@@ -875,6 +866,13 @@ @implementation DDLogMessage
 #define USE_DISPATCH_CURRENT_QUEUE_LABEL YES
 #define USE_DISPATCH_GET_CURRENT_QUEUE   YES
 
+#elif TARGET_OS_TV
```

Comment:
Maybe it would be cool better to merge these conditions as `#if TARGET_OS_IOS || TARGET_OS_WATCH`.